.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -2,6 +2,12 @@ name: Bug Report
 description: Report a bug in Quarkus
 labels: kind/bug
 body:
+  - type: markdown
+    attributes:
+      value: |
+        **To report a security vulnerability, please send an email to `security at quarkus dot io` with all the details.**
```

Comment:
How about making it clickable? 

```suggestion
        **To report a security vulnerability, please [send an email to `security at quarkus dot io`](mailto:security@quarkus.io) with all the details.**
```

Reply:
Do you think it would be a problem to use the email format here (`s/ at /@` and `s/ dot /\.`)? It sucks if you intend to copy/paste 😝

Reply:
Well, I have my personal email in my Git commits and I don't remember being bothered with spam because of that 😄

Reply:
I think it's a bit different as the git commits are a bit more hidden than the issue form. But we can change it if everybody agrees.

@maxandersen WDYT?

Reply:
Im fine. Not sure such Visual tricks stops spammers anyway.